frontend/src/bundles/common/components/modal/modal.tsx

```diff
@@ -9,11 +9,21 @@ import { type ModalProps } from '~/bundles/common/types/types.js';
 
 import styles from './styles.module.css';
 
-const Modal: React.FC<ModalProps> = ({ children, ...properties }) => {
+type Properties = {
+    modalReference?: React.RefObject<HTMLElement> | undefined;
```

Comment:
Using question mark is the same as specifying the type as undefined. 
```suggestion
    modalReference?: React.RefObject<HTMLElement>;
```